CHANGELOG.md

```diff
@@ -11,9 +11,12 @@
   * `cross: (Array[X], Array[Y]) => Array[Pair[X, Y]]` - create every possible pair from the two input arrays and return them all as WDL pairs
   * `transpose: (Array[Array[X]]) => Array[Array[X]]` compute the matrix transpose for a 2D array. Assumes each inner array has the same length.
 * By default, `system.abort-jobs-on-terminate` is false when running `java -jar cromwell.jar server`, and true when running `java -jar cromwell.jar run <wdl> <inputs>`.
-* Enable WDL imports when running in Single Workflow Runner Mode.
+* Enable WDL imports when running in Single Workflow Runner Mode as well as Server Mode
 * Support for sub workflows
+* Corrected file globbing in JES to be consistent. Additionally, file globbing in JES now uses bash-style glob syntax instead of python style glob syntax
```

Comment:
How about `Corrected file globbing in JES to correctly report all generated files. Additionally.....`